cosmwasm/submit-proposal.js

```diff
@@ -161,6 +161,41 @@ const execute = async (client, wallet, config, options) => {
     await callSubmitProposal(client, wallet, config, options, proposal);
 };
 
+const registerItsChain = async (client, wallet, config, options) => {
+    const chains = options.chains.map((chain) => {
+        const chainConfig = getChainConfig(config, chain);
+
+        if (!chainConfig.maxUintBits) {
```

Comment:
if `chain.chainType` is `evm`, default these two params to `uint256_max` and `u8_max` for simpler EVM operations (and set them in the chain ITS contract config)

Reply:
update README accordingly